packages/circuit-ui/components/NotificationToast/NotificationToast.docs.mdx

```diff
@@ -63,40 +58,35 @@ export function App({}) {
 
 ## Accessibility
 
-By their nature as small quick messages, or interruptions, toasts map to [live regions](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions).
-Live region must always exist on the page at render, before the toast is generated.
+By their nature as status messages, toasts are rendered inside a [live region](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) using `role="status"` and `aria-live="polite"`.
 
-Toast messages should not contain any interactive elements.
-Screen readers will announce raw text within the toast as it appears, without structure and without moving focus to the toast element.
-For example, if a toast read “You successfully updated your data” followed by an "Undo" button, screen readers would announce “You successfully updated your data. Undo”, not allowing keyboard users to interact with the button.
-In case you need to provide an action button, use NotificationInline component.
-
-To match the content, the role and aria-live level is adapted. The role is set to `"status"`, and it has `aria-live="polite"` meaning the screen reader will wait before announcing the update.
+A live region must always exist in the DOM _before_ content gets added to it, to ensure that additions are properly announced. In Circuit UI, this is handled by the `ToastProvider`.
 
 ### Best practices
 
-This component adheres to the following principles from the Web Content Accessibility Guidelines (WCAG):
+#### Toasts shouldn't have interactive content
+
+Toast messages should not contain any interactive elements. Screen readers will announce raw text within the toast as it appears, without structure and without moving focus to the toast element.
+
+For example, if a toast reads “You successfully updated your data" followed by an "Undo" button, screen readers will announce "You successfully updated your data. Undo", but wouldn't allow keyboard users to interact with the button because it will never receive focus.
+
+If you need a notification with an action button, use the [`NotificationInline`](Notifications/NotificationInline) component instead.
 
-#### Toast duration
+#### Toasts should be visible for at least 6 seconds
 
-The auto-dimiss duration of the toast is adjustable.
-Based on how fast the [average American reads](https://sheribyrnehaber.medium.com/designing-toast-messages-for-accessibility-fb610ac364be), a good lenght of time to keep the toast message up is 5 seconds plus 1 extra second for every 120 words.
-The shortest default that should be used as a best practice is 6 seconds.
+The auto-dismiss duration of the toast is adjustable.
 
-#### Consistent identification
+Based on how fast the [average American reads](https://sheribyrnehaber.medium.com/designing-toast-messages-for-accessibility-fb610ac364be), a good length of time to keep the toast message up is 5 seconds plus 1 extra second for every 120 words.
```

Comment:
> plus 1 extra second for every 120 words.

That seems very little! Who reads 120 words per second?! A quick Google search shows that the average reading speed is 250 words per minute.

Reply:
I'm pretty sure it should be 120 characters 😅 120 words should never ever be on a toast. 

I'm not sure where exactly the author of that article got their numbers, but I think I prefer the thinking behind this [UX StackExchange answer to "How long should a temporary notification (toast) appear?"](https://ux.stackexchange.com/a/85898/), though I think we can keep a longer default duration (especially since toasts can also be dismissed manually in our case).

But a key part of all timing questions for toasts would actually be making 100% sure that toasts are _short_. Long text should be in an inline notification.

Reply:
I think I'll merge this for now (this PR doesn't change the toast duration or duration docs) and rethink it in the next couple of days